.github/workflows/community-releases.yml

```diff
@@ -0,0 +1,98 @@
+name: Docker Community Release
```

Comment:
```suggestion
name: Prepare Self-hosted Release
```

Reply:
Let's also name it `prepare-self-hosted-release.yml` to match the prepare cloud release naming.